Remotion/Web/Core/Res/HTML/SmartPage.js

```diff
@@ -1176,11 +1176,20 @@ function SmartPage_Context(
       NextSubmitState : null
     };
     if (_submitState == null)
+    {
       _submitState = submitState;
-    else if (_submitState.IsAutoPostback && !isAutoPostback)
-      _submitState.NextSubmitState = submitState;
-    else
-      _submitState.NextSubmitState = { CancelSubmit: true, Submitter: null };
+    }
+    else if (_submitState.NextSubmitState == null)
+    {
+      if (_submitState.IsAutoPostback && !isAutoPostback)
+      {
+        _submitState.NextSubmitState = submitState;
+      }
+      else
+      {
+        _submitState.NextSubmitState = { CancelSubmit: true, Submitter: null };
+      }
+    }

```

Comment:
Please add an explation to the PR why this solution works. 

Originally, we've always overridden _submitState.NextSubmitState. Then, we've killed a second submits by overriding with CancelSubmit=true. Now, if a second submit is already queued, we just leave that one.

(right now, I don't want to re-think this myself)

Reply:
Consider the following postback sequences:

### `Button`

- Set `_submitState`, nothing to queue here.

### `Autopostback`, `Button`

- Set `_submitState` for the autopostback.
- Set `_submitState.NextSubmitState` for the button click.

### `Button` (1), `Button` (2)

- Set `_submitState` for the first button.
- Set `_submitState.NextSubmitState` to `{ CancelSubmit: true, Submitter: null }` for the second button.

### `Autopostback`, `Link`, `Button`

- Set `_submitState` for the autopostback.
- Set `_submitState.NextSubmitState` for the link click.
- Button handling
  - Before:
    - Overwrite `_submitState.NextSubmitState`.
  - After:
    - Do nothing, there's already a queued postback.